lists/pe/RUC.json

```diff
@@ -0,0 +1,51 @@
+{
+  "name": {
+    "en": "National Superintendency of Customs and Tax Administration (Peru)",
+    "local": "Superintendencia Nacional de Aduanas y de Administración Tributaria (Perú)"
+  },
+  "url": "https://www.sunat.gob.pe/",
+  "description": {
+    "en": "The 'Registro Único del Contribuyente' (RUC) number is an 11-digit tax ID number for any natural or legal person (such as a company) in Peru, and is provided and maintained by SUNAT. The SUNAT, according to its creation Law No. 24829, is a specialized technical agency, attached to the Ministry of Economy and Finance. Its main purpose is to administer the taxes of the national government and the tax and non-tax concepts entrusted to it by Law or according to the inter-institutional agreements entered into, providing the resources required for fiscal solvency and macroeconomic stability; ensuring the correct application of the regulations governing the matter and combating tax and customs crimes in accordance with its powers."
```

Comment:
If we remove 'Superintendencia Nacional de Aduanas y de Administración Tributaria' from the name of this list, then it should be added to the description here the first time 'SUNAT' is mentioned.